src/scene/frame-graph.js

```diff
@@ -90,6 +90,41 @@ class FrameGraph {
             }
         }
 
+        // merge passes if possible
+        for (let i = 0; i < renderPasses.length - 1; i++) {
+            const firstPass = renderPasses[i];
+            const firstRT = firstPass.renderTarget;
+            const secondPass = renderPasses[i + 1];
+            const secondRT = secondPass.renderTarget;
+
+            // if the render targets are different, we can't merge the passes
+            // also only merge passes that have a render target
+            if (firstRT !== secondRT || firstRT === undefined) {
```

Comment:
`!firstRT` ?

Reply:
nope, a NULL is valid case, just not undefined. null means rendering to backbuffer.